pySDC/projects/Monodomain/problem_classes/TestODE.py

```diff
@@ -0,0 +1,116 @@
+import logging
+import numpy as np
+from pySDC.core.Problem import ptype
+from pySDC.core.Common import RegisterParams
+from pySDC.implementations.datatype_classes.mesh import mesh
+from pySDC.projects.Monodomain.datatype_classes.my_mesh import imexexp_mesh
+
+
+""" 
+Here we define the problems classes for the multirate Dahlquist test equation y'=lambda_I*y + lambda_E*y + lambda_e*y
+Things are done so that it is compatible witht the sweepers. 
+"""
+
+
+class Parabolic(RegisterParams):
+    def __init__(self, **problem_params):
+        self._makeAttributeAndRegister(*problem_params.keys(), localVars=problem_params, readOnly=True)
+        self.shape = (1,)
+        self.init = ((1,), None, np.dtype("float64"))
+
+
+class TestODE(ptype):
+    def __init__(self, **problem_params):
+        self.logger = logging.getLogger("step")
+
+        self.parabolic = Parabolic(**problem_params)
+        self.size = 1  # one state variable
+        self.init = ((self.size, *self.parabolic.init[0]), self.parabolic.init[1], self.parabolic.init[2])
+
+        # invoke super init
+        super(TestODE, self).__init__(self.init)
+        # store all problem params dictionary values as attributes
+        self._makeAttributeAndRegister(*problem_params.keys(), localVars=problem_params, readOnly=True)
+
+        # initial and end time
+        self.t0 = 0.0
+        self.Tend = 1.0 if self.end_time < 0.0 else self.end_time
+
+        # set lambdas, if not provided by user
+        if not hasattr(self, 'lmbda_laplacian'):
+            self.lmbda_laplacian = -5.0
+        if not hasattr(self, 'lmbda_gating'):
+            self.lmbda_gating = -10.0
+        if not hasattr(self, 'lmbda_others'):
+            self.lmbda_others = -1.0
+
+        self.dtype_u = mesh
```

Comment:
In other problem classes these are class attributes in pySDC. Previously, there was some instance-specific logic, but if there isn't, it's better to make this a class attribute for constituency.

Reply:
agree, fixed